src/instructies-invoerder.md

```diff
@@ -31,76 +21,108 @@ Je kunt in Abacus een aantal sneltoetsen gebruiken waarmee je handig naar andere
+### 3.1. Verkiezing en stembureau selecteren
 
-![Waarschuwingen bij Toegelaten kiezers en uitgebrachte stemmen](./img/invoeren-kiezers-stemmen-waarschuwingen.png)
+Selecteer eerst de verkiezing waarvoor je stemmen wilt invoeren. Hier zie je ook wat de status van de verkiezing is.
 
-### Verschillen
+![Verkiezing selecteren](./img/invoeren-verkiezing-selecteren.png)
 
-In rubriek 3 op het procesverbaal is aangegeven of er verschillen zijn tussen het aantal toegelaten kiezers en het aantal uitgebrachte stemmen, en wat de verklaringen hiervoor zijn. Neem de aantallen uit het proces-verbaal over in de velden I tot en met O.
+Selecteer nu het stembureau:
 
-**Let op:** op het proces-verbaal zijn deze velden niet voorzien van letters. Kijk dus goed of je de cijfers juist overneemt.
+- Voer het nummer in van het stembureau dat je wilt invoeren. Het nummer vind je op het proces-verbaal.
+- Weet je het nummer niet, klik dan onderaan de pagina op **Toon de lijst met nog in te vullen stembureaus** en kies vervolgens het juiste stembureau.
 
-![Verschillen tussen toegelaten kiezers en uitgebrachte stemmen](./img/invoeren-verschillen.png)
+![Stembureau kiezen](./img/invoeren-stembureau-kiezen.png)
 
-### Lijsten
+### 3.2. Extra onderzoek
 
-Nu ga je invoeren hoeveel stemmen zijn geteld voor elke kandidaat op elke lijst.
+Op pagina 1 van het papieren formulier is aangegeven of er extra onderzoek is uitgevoerd.
 
-Neem de cijfers over zoals ze in het proces-verbaal staan. Lege velden en velden met een nul op het proces-verbaal kun je ook leeglaten in Abacus.
+- Neem de vinkjes over zoals ze in het proces-verbaal staan en klik op **Volgende**.
+- Je ziet een fout als er twee vinkjes zijn gezet bij dezelfde vraag.
+- Als de vragen op het papier niet zijn ingevuld, dan is er geen extra onderzoek uitgevoerd en kun je direct doorgaan naar de volgende pagina.
 
-**Let op:** Op het proces-verbaal staan ook velden voor het subtotaal van elke kolom. Deze kun je negeren.
+![Extra onderzoek](./img/invoeren-extra-onderzoek.png)
 
-Als je bij de kandidaten alle aantallen hebt ingevuld en je tot het totaal alleen maar lege velden ziet, ga je met `shift ⇧`+`pijl omlaag ↓` direct naar het invoerveld voor het totaal.
+### 3.3. Verschillen met stembureau
 
-Als je klaar bent met het invoeren van de lijst, klik je op **Volgende** of druk je op `shift ⇧`+`enter ⏎` om naar de volgende lijst te gaan.
+Op pagina 2 van het papieren formulier is aangegeven of er verschillen zijn met de telresultaten van het stembureau.
 
-**Let op:** Er kan een foutmelding worden weergegeven als de aantallen niet met elkaar kloppen. Controleer of je de aantallen goed hebt overgenomen. Als de aantallen overeenkomen met het proces-verbaal en je de fout nog steeds ziet, overleg dan met je coördinator.
+- Neem de vinkjes over zoals ze in het proces-verbaal staan en klik op **Volgende**.
+- Je ziet een fout als er twee vinkjes staan bij dezelfde vraag of als er niets is ingevuld.
```

Comment:
Zie vorige comment. Ik zou dit generieker maken, en alleen zeggen dat Abacus je instructies geeft bij onlogische invoer of fouten op het papieren proces-verbaal. 

Vraag is dan of we dit specifieke screenshot erbji moeten doen, of liever de empty state / happy flow